rclpy/rclpy/timer.py

```diff
@@ -35,6 +36,26 @@ def timer_period_ns(self, value):
         _rclpy.rclpy_change_timer_period(self.timer_handle, val)
         self._timer_period_ns = val
 
+    @property
+    def is_canceled(self):
+        val = _rclpy.rclpy_is_timer_canceled(self.timer_handle)
+        self._is_canceled = True
+        return val
+
+    @is_canceled.setter
+    def is_canceled(self, value):
+        raise NotImplementedError('canceled is set by calling the `timer_cancel` function')
```

Comment:
I went back and forth because I wanted hide some if the logic inside Python but it doesn't seem to add much value so I went back to just bind directly all functions and provide a Python method for each of the C functions: see https://github.com/ros2/rclpy/pull/54/commits/25c1aabb2c58a82e4a785b6a4d09eb07629cc201